### DISCUSSION TITLE
It doesn't exist in Startup Windows! It won't run automatically!

### DISCUSSION THREAD

**Discussion by naviddavami** [HerMajestyDrMona/Windows11DragAndDropToTaskbarFix#42] [Q&A] [answered]
Hi guys
few months ago I write this text in CMD:
![OPlSSDAqiRA9NGYtIRsgIzVCwtVziIpxy1KL9L5h77GmEaANsST3Mk8e8U7G7N8w](https://user-images.githubusercontent.com/68032252/147079721-ed81aec8-1ff4-45b4-83bd-d33f6a80dbb3.png)

After that , I install (Windows11DragAndDropToTaskbarFix_2.1.0.0) and it run without any issues but when I restart my system, it won't run automatically !!!
Also it doesn't exist in Startup Windows (Screenshot Below):
![CXxxiK05Qw8KJO1VCU5D50eyNPeDYcd311tDgSAxk9CTo6KQSHUPhYjvjnzdP69l](https://user-images.githubusercontent.com/68032252/147080952-f652bb17-96fc-4ec3-9e6c-2652fe407941.png)

How can I fix this problem ?
Can you help me ?

**Comment by HerMajestyDrMona** [comment, marked as answer]
Hi. Did you read the [CONFIGURATION](https://github.com/HerMajestyDrMona/Windows11DragAndDropToTaskbarFix/blob/main/CONFIGURATION.md) page?

All you need to do is clicking on the Mona Lisa Metan Fox tray icon, and then `Configure...` button. It will create the configuration file with autostart enabled by default.

It was intentionally disabled by default, because some antivirus programs were finding it as malicious behaviour, especially when many users started this program directly from the Downloads folder.

**Reply by naviddavami** [reply]
That's right. I clicked on the Mona Lisa Metan Fox tray icon and autostart was enable:
![PQzH5JdWmD6kKORkujYkQZIgRh6iiM3y7Vb1aesMiPxS9k6TtW9ERilePIMXVIru](https://user-images.githubusercontent.com/68032252/147082026-384ba306-12b4-4951-a85e-43bf09948505.png)

My antivirus doesn't any problem with this program. How to change enable by default ?

**Reply by HerMajestyDrMona** [reply]
It looks fine. Can you please check if registry key: `HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Run\Windows11DragAndDropToTaskbarPartialFix` is created?

Usually when there's some restriction in writing to the registry, you could quit the program and then run it as administrator 1 time. Then it should be able to create the registry key with no limitations.

**Reply by naviddavami** [reply]
I checked registry key but there wasn't any result of this program!!!
then I ran as administrator but it didn't create registry key.
![2hxWQLdzUJsAmUXQQzxzzp69GKqylZ0GBx8SecMYLaZyUskGMcAhR7GYjWRurR3t](https://user-images.githubusercontent.com/68032252/147084506-283bfd2c-a5f6-4052-ac4b-70c7bbc76aea.png)

**Reply by HerMajestyDrMona** [reply]
That's strange. I wonder if the configuration file is actually read correctly. Can you try to enable the console window:
`ShowConsoleWindowOnStartup=1`

And see if it correctly shows up on program start?

Also, are you sure you quit the program before starting it as administrator? When it's already running the new instance automatically quits, so might not continue with autostart check.

**Reply by naviddavami** [reply]
yesss. done :)
when I enable console windows , program showed up...
Thank you very much.

**Reply by HerMajestyDrMona** [reply]
Thanks for feedback. So the configuration loading works, and even with administration permissions it won't create registry keys... That's strange. I don't know how to reproduce this issue or how to fix it unfortunately.
